helpers/json/response_json.go

```diff
@@ -0,0 +1,147 @@
+package jsonhelpers
+
+import (
+	"net/http"
+
+	"github.com/markgravity/golang-ic/helpers"
+	"github.com/markgravity/golang-ic/lib/validators"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-playground/validator/v10"
+	"github.com/google/jsonapi"
+	"github.com/samber/lo"
+)
+
+const (
+	codeBadRequestError          string = "bad_request_error"
+	codeForbiddenError           string = "forbidden_error"
+	codeInternalServerError      string = "internal_server_error"
+	codeNotFoundError            string = "not_found_error"
+	codeUnauthorizedError        string = "unauthorized_error"
+	codeUnprocessableEntityError string = "unprocessable_entity_error"
+	codeServiceUnavailable       string = "service_unavailable_error"
+	codeValidationError          string = "validation_error"
+)
+
+func RenderError(ctx *gin.Context, statusCode int, errMessage string, errorCode string) {
```

Comment:
```suggestion
func renderError(ctx *gin.Context, statusCode int, errMessage string, errorCode string) {
```

It seems that at the moment you don't need to expose this function

Reply:
@malparty Please ignore it as it's the code base of gulf project, we will use it later 🙏